File: 프로그래머스/unrated/134240. 푸드 파이트 대회/푸드 파이트 대회.java
class Solution {
    public String solution(int[] food) {
        String answer = "";
        for (int i = 1; i < food.length; i++) {
            if (food[i] > 1) {
                for (int j = 0; j < food[i]/2; j++) {
                    answer += Integer.toString(i);
                }
            }
        }
        String half = answer;
        answer += "0";
        StringBuffer sb = new StringBuffer(half);
        String reversedStr = sb.reverse().toString();
        answer += reversedStr;
        return answer;
    }
}

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    answer = ''
    for i in range(len(food)):
        for j in range(food[i]//2):
            answer += str(i)
    answer += "0"
    for i in range(len(answer)-2, -1, -1):
        answer += str(answer[i])
    return answer